src/constants/cart/index.js

```diff
@@ -0,0 +1,7 @@
+export const CART_STORAGE = 'cart';
```

Comment:
specify what is this for?

Reply:
renamed to CART_STORAGE_KEY for specificity.